[2025-07-30T02:39:20.017+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-07-30T02:39:20.030+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fraud_detection_training.validate_environment manual__2025-07-30T02:39:19.014636+00:00 [queued]>
[2025-07-30T02:39:20.035+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fraud_detection_training.validate_environment manual__2025-07-30T02:39:19.014636+00:00 [queued]>
[2025-07-30T02:39:20.036+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-07-30T02:39:20.048+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): validate_environment> on 2025-07-30 02:39:19.014636+00:00
[2025-07-30T02:39:20.053+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=159) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-07-30T02:39:20.054+0000] {standard_task_runner.py:72} INFO - Started process 160 to run task
[2025-07-30T02:39:20.054+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'fraud_detection_training', 'validate_environment', 'manual__2025-07-30T02:39:19.014636+00:00', '--job-id', '83', '--raw', '--subdir', 'DAGS_FOLDER/fraud_detection_training_dag.py', '--cfg-path', '/tmp/tmpff6zus1q']
[2025-07-30T02:39:20.055+0000] {standard_task_runner.py:105} INFO - Job 83: Subtask validate_environment
[2025-07-30T02:39:20.095+0000] {task_command.py:467} INFO - Running <TaskInstance: fraud_detection_training.validate_environment manual__2025-07-30T02:39:19.014636+00:00 [running]> on host 75f9886374a6
[2025-07-30T02:39:20.164+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='kanewyp' AIRFLOW_CTX_DAG_ID='fraud_detection_training' AIRFLOW_CTX_TASK_ID='validate_environment' AIRFLOW_CTX_EXECUTION_DATE='2025-07-30T02:39:19.014636+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-07-30T02:39:19.014636+00:00'
[2025-07-30T02:39:20.165+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-07-30T02:39:20.180+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-07-30T02:39:20.181+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', '\n        echo "Validating environment..."\n        test -f /app/config.yaml &&\n        test -f /app/.env &&\n        echo "Environment validation successful."\n        ']
[2025-07-30T02:39:20.192+0000] {subprocess.py:99} INFO - Output:
[2025-07-30T02:39:20.193+0000] {subprocess.py:106} INFO - Validating environment...
[2025-07-30T02:39:20.194+0000] {subprocess.py:106} INFO - Environment validation successful.
[2025-07-30T02:39:20.194+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-07-30T02:39:20.214+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-07-30T02:39:20.214+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=fraud_detection_training, task_id=validate_environment, run_id=manual__2025-07-30T02:39:19.014636+00:00, execution_date=20250730T023919, start_date=20250730T023920, end_date=20250730T023920
[2025-07-30T02:39:20.270+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-07-30T02:39:20.293+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-07-30T02:39:20.294+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
